docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/personalizing-collections.md

```diff
@@ -8,20 +8,28 @@ By default, any user can see all the Collection's content and is part of *Anyone
 To customize a Collection using Segments, you must define the Segments first. For more information, see [Creating and Managing User Segments](../segmentation/creating-and-managing-user-segments.md).
 ```
 
-![Collections are available for anyone, by default](./personalizing-collections/images/02.png)
+![Collections are available for anyone, by default](./personalizing-collections/images/01.png)
 
 Consider the following example. You want to increase sales in your online Kitchenware Store by offering exclusive promotions to registered users. You create a Collection including promotions for registered users and news about your products. You want all users to see the news about your products, but you want to restrict the promotions to registered users. In this example, you can create a new Segment for registered users, and link the Segment to a new Personalized Variation that filters the promotional content.
 
 ```{tip}
 You can create multiple Personalized Variations for a Collection, and edit or delete them as needed. You can also edit the *Anyone* Personalized Variation, but you cannot delete it.
 ```
 
+## Prioritizing Variations
```

Comment:
I think about this two things:
- The highest priority is the position on top of the list
- The default variation, which obviously matches every user, can be placed at will, hiding the segments below.

Do you think is useful to mention that here?